wagn/features/step_definitions/wagn_steps.rb

```diff
@@ -342,6 +348,12 @@ def signed_in_as username
   page.has_css?(".diff-#{css_class}", text: text)
 end
 
+Then /^I should see css class "([^\"]*)" within "(.*)"$/ do |css_class, selector|
```

Comment:
Can we exempt the line restriction in cucumber? It seems regular expression does not support multi-lines. I am not sure.